src/server.rs

```diff
@@ -729,13 +718,6 @@ pub enum ServerSet {
     ///
     /// Runs in [`PreUpdate`].
     ReceivePackets,
-    /// Systems that emit [`ServerEvent`].
-    ///
-    /// The messaging backend should convert its own connection events into [`ServerEvents`](ServerEvent)
-    /// in this set.
-    ///
-    /// Runs in [`PreUpdate`].
-    SendEvents,
```

Comment:
I think it's useful to keep this. Since most backends need to put these systems *somewhere*.

Reply:
Good point! Restored. But I decided to rename it into `TriggerServerEvents` for clarity and updated the description. Please, let me know if it's is clear enough.